packages/module/src/WarningModal/WarningModal.tsx

```diff
@@ -19,37 +25,55 @@ const WarningModal: React.FunctionComponent<WarningModalProps> = ({
   cancelButtonLabel = 'Cancel',
   variant = ModalVariant.small,
   titleIconVariant = 'warning',
+  withCheckbox = false,
+  confirmCheckMessage='',
```

Comment:
We should provide a default message - this is, what we use in RBAC remove modals
```suggestion
  confirmCheckMessage='I understand that this action cannot be undone.',
```